uptane-standard.md

```diff
@@ -822,7 +833,7 @@ In order to perform full verification, an ECU SHALL perform the following steps:
 11. Verify that Targets metadata from the Director and Image repositories match. A primary ECU MUST perform this check on metadata for all images listed in the Targets metadata file from the Director repository downloaded in step 6. A secondary ECU MAY elect to perform this check only on the metadata for the image it will install. (That is, the target metadata from the Director that contains the ECU identifier of the current ECU.) To check that the metadata for an image matches, complete the following procedure:
     1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
     2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
+        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted or encrypted image are the same in both sets of metadata. Note: the primary is responsible for validating encrypted images and associated metadata. The target ECU (primary or secondary) is responsible for validating the unencrypted image and associated metadata.
```

Comment:
Again, I don't see how this is possible. The images repository doesn't know anything about encrypted images; they only live in the Director repository. It would be quite difficult to get that metadata to match in that case.

Reply:
Thanks for the comment, Patrick.

This was discussed in-length on a call 1 or 2 weeks ago?
Essentially, the comment "image repository does not have encrypted images" is one that will not be 100% true in production.
Ira gave a very good example of companies wanting to protect IP and ensuring images are encrypted until they reach their **target** ECU.
To account for this, I put this language in the PR.
If you have a proposal for better language to include, please let me know.

Reply:
In principle, I have no objection to the language here. If others agree that it doesn't conflict with the encryption language described elsewhere, then I have no problem with it.

Reply:
Supporting encrypted images is certainly necessary in production. For the encrypted data usecase, verification of unencrypted data may only be possible on the target ECU due to knowledge of the encryption keys. One very important consideration is the order of operations for encryption / verification. We need to be very clear in what steps the encryption happens. In my opinion, we need to require validation of the unencrypted data on the target ECU to ensure end to end authenticity and integrity.